packet_injector/client.go

```diff
@@ -247,17 +253,17 @@ func (pc *PacketInjectorClient) OnSwitchToSlave() {
 
 func (pc *PacketInjectorClient) onAPIWatcherEvent(action string, id string, resource types.Resource) {
 	logging.GetLogger().Debugf("New watcher event %s for %s", action, id)
-	ppr := resource.(*types.PacketParamsReq)
+	ppr := resource.(*types.PacketInjection)
```

Comment:
nit: how about some other name instead of ppr?

Reply:
done